### ISSUE TITLE
Feat : To create the social icons tab in the Home page 

### ISSUE THREAD

**Issue by DhairyaMajmudar** [IIITians-Network-23/IIITiansNetwork#10]
Create the icons tab in the home page as mentioned in the Figma file  developer have to add relevant opening and closing animation in it

**Comment by DhairyaMajmudar** [comment]
@adysingh5711 would you like to take this up

**Comment by adysingh5711** [comment]
Yeah sure but will require more details on it and the Figma file as new to it :)

**Comment by DhairyaMajmudar** [comment]
Sure you can take help from #2 for that
Thanks !
